sbin/health-checker.in

```diff
@@ -71,77 +58,77 @@ telem_send_record()
     fi
 }
 
-save_working_snapshot()
-{
-    set_btrfs_id
-
-    if [ ${BTRFS_ID_DEFAULT} -eq ${BTRFS_ID_CURRENT} ]; then
-	echo "LAST_WORKING_BTRFS_ID=${BTRFS_ID_DEFAULT}" > $STATE_FILE
-	echo "LAST_WORKING_SNAPSHOT=${SNAPSHOT_DEFAULT}" >> $STATE_FILE
-    fi
-}
-
-rollback()
-{
-    . ${STATE_FILE}
-    mount -o remount,rw /.snapshots
-    btrfs subvolume set-default ${LAST_WORKING_BTRFS_ID} /.snapshots
-    if [ $? -ne 0 ]; then
-        create_log user.crit "ERROR: btrfs set-default $BTRFS_ID_DEFAULT failed!"
-	telem_send_payload 1
-        exit 1
+systemd-bless-boot() {
+    if [ -x /usr/lib/systemd/systemd-bless-boot ]; then
```

Comment:
Can this happen resp. is it supposed to happen that this does not exist? If yes, do the callers handle it properly that nothing happens?